docs/source/conf.py

```diff
@@ -244,4 +244,7 @@ def setup(app):
         'enable_auto_doc_ref': False,
     }, True)
     app.add_transform(AutoStructify)
-    app.add_stylesheet('custom.css')  # may also be an URL
+    try:
+        app.add_stylesheet('custom.css')  # may also be an URL
+    except AttributeError:
+        app.add_css_file('custom.css')  # may also be an URL
```

Comment:
Just as an FYI, is there one of these methods we should be preferring over the other? I don't really understand this line (it's not a blocker to  👍 approving the PR but it might help the next person looking at the code, too).

Reply:
We should be using `app.add_css_file('custom.css')` I guess this broke due to a newer sphinx version coming in. I can move to only use that one :), but I was not sure if some of the CI tests would break.

Let me try and we can check.

Thanks!